lib/elixir/lib/module/parallel_checker.ex

```diff
@@ -158,8 +159,50 @@ defmodule Module.ParallelChecker do
     :gen_server.call(server, {:unlock, module})
   end
 
+  defp prepare_modules(modules) do
+    Enum.map(modules, fn {map, binary} ->
+      {:ok, _module, chunks} = :beam_lib.all_chunks(binary)
+      map = fill_missing_map_fields(map, chunks)
+      {map, chunks}
+    end)
+  end
+
+  defp fill_missing_map_fields(map, chunks) do
+    map
+    |> debug_info_to_map(chunks)
+    |> checker_chunk_to_map(chunks)
+  end
+
+  defp debug_info_to_map(%{definitions: nil, file: nil} = map, chunks) do
+    with {'Dbgi', debug_info} <- :lists.keyfind('Dbgi', 1, chunks),
+         {:debug_info_v1, backend, data} <- :erlang.binary_to_term(debug_info),
+         {:ok, info} <- backend.debug_info(:elixir_v1, map.module, data, []) do
+      %{map | definitions: info.definitions, file: info.relative_file}
+    else
+      _ -> %{map | definitions: []}
```

Comment:
In this case, wouldn't it be better to skip the module altogether?

Reply:
For example, if we write a ExCk for this, it would likely be wrong.

Reply:
Yeah, let's skip it 👍